### PR TITLE
RavenDB-12529: Added select all and height limit to indexes multiselect, added actions multiselect

### PR DESCRIPTION
**Pull request by garayx**

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/database/indexes/indexErrors.ts [hunk 1]
```diff
@@ -196,6 +229,26 @@ class indexErrors extends viewModelBase {
...
+    private extractActionNamesAndCount(indexErrors: Raven.Client.Documents.Indexes.IndexErrors[]): Array<indexActionAndCount> {
+        const mappedItems = _.flatMap(indexErrors,
+            value => {
+                return value.Errors.map(x => ({
+                    actionName: x.Action,
+                    count: 1
+                } as indexActionAndCount));
+            });
+
+        return mappedItems.reduce((result: indexActionAndCount[], next: indexActionAndCount) => {
```
Comment (grisha-kotler) [inline src/Raven.Studio/typescript/viewmodels/database/indexes/indexErrors.ts]: Can we sort the result by actionName?

Comment (grisha-kotler) [reply src/Raven.Studio/typescript/viewmodels/database/indexes/indexErrors.ts]: Sorting is also needed for the index name

**Review by grisha-kotler — CHANGES_REQUESTED** [review]

**Review by ml054 — APPROVED** [review]
LGTM. Please fix 2 notes from Grisha.

File: src/Raven.Studio/typescript/viewmodels/database/indexes/indexErrors.ts (hunk 1 above)
Comment (garayx) [reply src/Raven.Studio/typescript/viewmodels/database/indexes/indexErrors.ts]: done

File: src/Raven.Studio/typescript/viewmodels/database/indexes/indexErrors.ts [hunk 2]
```diff
@@ -188,12 +221,42 @@ class indexErrors extends viewModelBase {
...
     private extractIndexNamesAndCount(indexErrors: Raven.Client.Documents.Indexes.IndexErrors[]): Array<indexNameAndCount> {
-        return indexErrors.filter(error => error.Errors.length > 0).map(errors => {
+        const array = indexErrors.filter(error => error.Errors.length > 0).map(errors => {
             return {
                 indexName: errors.Name,
                 count: errors.Errors.length
             }
         });
+
+        return array.sort((a, b) => this.sortStrings(a.indexName, b.indexName));
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/indexes/indexErrors.ts]: let's simply use lodash:
_.sortBy(array, x => x.indexName)

Comment (garayx) [reply src/Raven.Studio/typescript/viewmodels/database/indexes/indexErrors.ts]: done

Comment (ppekrol) [reply src/Raven.Studio/typescript/viewmodels/database/indexes/indexErrors.ts]: Done

**Pull request merged**